### PR TITLE
Update certificate-practice-statement.md

### PR DESCRIPTION
**Pull request by jochemoosterlee-cb**
Created first "page"

### CODE REVIEW

File: certificate-practice-statement.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Certification Practice Statement
+# Certification Practice Statement DRAFT
```
Comment (jochem-oosterlee) [inline certificate-practice-statement.md:1]: This shouldn't be added in a release version.

Comment (jochem-oosterlee) [inline certificate-practice-statement.md:1]: This shouldn't be added to a release version

**Review by jochem-oosterlee — CHANGES_REQUESTED** [review]
See comment in certificate-pratice-statement.md

**Comment by jochemoosterlee-cb** [comment]
Made changes as requested

**Review by jochem-oosterlee — APPROVED** [review]
Good to go!

**Pull request merged**